components/02-elements/swatch/swatch.config.js

```diff
@@ -27,51 +24,33 @@ module.exports = {
     ]
   },
   variants: [
-    {
-      name: 'small',
-      context: {
-        class: 'swatch--small'
-      }
-    },
     {
       name: 'image',
       context: {
-        class: 'swatch--image',
-        heading: false,
-        options: [
-          {
-            attributes: 'style="background-image: url(../../images/swatch/swatch-orange.jpg);"'
-          },
-          {
-            attributes: 'style="background-image: url(../../images/swatch/swatch-green.jpg);"'
-          },
-          {
-            class: 'swatch__option--active',
-            attributes: 'style="background-image: url(../../images/swatch/swatch-red.jpg);"'
-          },
-          {
-            attributes: 'style="background-image: url(../../images/swatch/swatch-white.jpg);"'
-          }
-        ]
-      }
-    },
-    {
-      name: 'small-image',
-      context: {
-        class: 'swatch--small swatch--image',
         heading: false,
+        wrapper: {
+          attributes: 'aria-label="Color" tabindex="0"',
+        },
         options: [
           {
+            color: 'Orange',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-orange.jpg);"'
           },
           {
+            color: 'Green',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-green.jpg);"'
           },
           {
-            class: 'swatch__option--active',
+            color: 'Red',
+            class: 'selected',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-red.jpg);"'
           },
           {
+            color: 'White',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-white.jpg);"'
           }
         ]
```

Comment:
I've updated the config with aria attributes, please implement it where needed (views, modules)